src/utils/wallet.js

```diff
@@ -293,6 +305,11 @@ const wallet = {
    * @inner
    */
   reloadData() {
+    if (!hathorLib.wallet.loaded()) {
```

Comment:
When will it be not loaded?

Reply:
We had a bug that sometimes we were resetting the wallet and losing connection to websocket, so we were trying to reload data without any data (because we had reset the wallet before).

This is a fix for this bug, however I have already made another fix on the lib (which is better because it fixes for all apps using the lib), so we can remove this code here.

Reply:
Removed this code